src/averageAges.js

```diff
@@ -15,11 +25,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const calculateCentury = num => Math.ceil(num / 100);
+
+  const men = people.filter(person =>
+    century
+      ? calculateCentury(person.died) === century && isMale(person)
```

Comment:
you can implement this `calculateCentury(person.died) === century` inside of the calculateCentury and return a boolean, just like isMale